python/src/DistFunc_doc.i.in

```diff
@@ -600,20 +701,357 @@ Returns
 -------
 p : float"
 
+
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::DistFunc::rNonCentralChiSquare
+"Realization of a  NonCentralChiSquare distribution.
+
+Parameters
+----------
+nu: float, :math:`\nu > 0`,
+    The :math:`\nu` parameter.
+lambda: float, :math:`\lambda \geq 0`,
+    The :math:`\lambda` parameter.
+size : int,
+    The number of realizations (optional).
+    
+    By default, size is equal to 1.
+
+Returns
+-------
+realization : float or a sequence of float if size is greater than 1.
+
+Notes
+-----
+The position parameter  :math:`\gamma` is equal to zero.
+
+The probability density function is defined as:
+
+.. math::
+
+    f_X(x) = \sum_{j=0}^{\infty} e^{-\lambda}
+                                 \frac{\lambda^j}{j!}p_{\chi^2(\nu + 2j)}(x),
+             \quad x \in [0; +\infty[
+
+where :math:`p_{\chi^2(q)}` is the probability density function of a
+:math:`\chi^2(q)` random variate.
+
+
+Examples
+--------
+>>> import openturns as ot
+>>> ot.RandomGenerator.SetSeed(0)
+>>> r = ot.DistFunc.rNonCentralChiSquare(2.5, 0.5)
+>>> r = ot.DistFunc.rNonCentralChiSquare(2.5, 0.5, 10)"
+
+
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::DistFunc::pNonCentralChiSquare
+"Cumulative density function  of a NonCentralChiSquare.
+
+Parameters
+----------
+nu: float, :math:`\nu > 0`,
+    The :math:`\nu` parameter.
+lambda: float, :math:`\lambda \geq 0`,
+    The :math:`\lambda` parameter.
+x: float, 
+    Location.
+tail: bool, default=False
+    Tail flag.
+    
+    If True, the complementary cdf is computed.
+precision: float,
+    The precision of the evaluation.
+maxInter: int,
+    The maximum numbe rof iterations of the algorithm.
+
+Returns
+-------
+cdf : float,
+    The cdf or the complementary cdf at x.
+
+Notes
+-----
+The position parameter  :math:`\gamma` is equal to zero.
+
+The probability density function is defined as:
+
+.. math::
+
+    f_X(x) = \sum_{j=0}^{\infty} e^{-\lambda}
+                                 \frac{\lambda^j}{j!}p_{\chi^2(\nu + 2j)}(x),
+             \quad x \in [0; +\infty[
+
+where :math:`p_{\chi^2(q)}` is the probability density function of a
+:math:`\chi^2(q)` random variate.
+
+We use Benton and Krishnamoorthy's algorithm described in [benton2003]_.
```

Comment:
there is no entry in python/doc/user_manual for DistFunc::pNonCentralChiSquare, so the citation is not taken into account

Reply:
Thanks a lot! I try...